src/coffee/run.coffee

```diff
@@ -244,7 +244,10 @@ module.exports = class
               importer.continueOnProblems = opts.continueOnProblems
               importer.changeState(publish, remove, filterFunction)
             .then (result) ->
-              _consoleWarnAllResults result
+              if result.length > 100
+                fs.writeFileSync opts.output, JSON.stringify(result, null, 2)
```

Comment:
It will throw an error.

Reply:
Ok, how about making sure this condition also checks if we have added the `-o` flag? Otherwise, this looks like a breaking change since it can cause new unexpected errors.

Reply:
Cool, thanks for adding the condition. Maybe we could then also increase the limit? Impex should be able to show 100 without any problems I think?

Reply:
I will suggest to leave it at default for this repo so that when `Util.inspect()` is removed in the future only the two remaining modules will need to be fixed.

Reply:
I still don't see a reason to why we want to remove `Util.inspect()` though? Could you please elaborate to why you want to make that change? 

I read the link you posted above and only saw this

>The util.inspect() method returns a string representation of object that is intended for debugging. The output of util.inspect may change at any time and should not be depended upon programmatically. 

Since we are only passing this string to `console.warn` and use it for debugging and logging the results of the app it should be fine to keep it. 

In other words, we don't depend on the output in any other way in the app as we just log it out.

Maybe I'm missing something, please let me know.

Reply:
I think we should avoid `Util.inspect()` because it doesn't provide a permanent solution to the problem of logging.

It solved the problem when it was implemented then and now the same logging problem arise and since it was stated in the `Node docs` that `SHOULD NOT BE DEPENDED ON PROGRAMMATICALLY` and  we are using in IMPEX(production) for logging, my suggestion would be to avoid it and find a permanent solution or alternatives.

Reply:
I see your point but I think you missed my line above.

> In other words, we don't depend on the output in any other way in the app as we just log it out.

In fact, even `console.warn` itself uses `util.inspect()`([ref](https://nodejs.org/api/console.html#console_console_error_data_args)).

You could even test it out yourself by setting the default option to it [like this](https://nodejs.org/api/util.html#util_util_inspect_defaultoptions) then only calling `console.warn`. It will have exactly the same effect.